modules/project-docs/pages/compatibility.adoc

```diff
@@ -80,6 +80,24 @@ M1 ARM  architecture is fully supported in the PHP SDK.
 AWS Amazon Graviton2, Apple M1 ARM processors, and ARMv8 on Ubuntu 20.04+ (from SDK 4.1).
 ****
 
+=== x86-32 Processor Support
+
+While the SDK can be compiled for 32-bit version of x86 platform, it is not
+generally tested and supported type of the deployment.
+
+PHP does not have platform-independent
+https://www.php.net/manual/en/language.types.integer.php[64-bit integer], and
+all numbers are always signed. These facts do not allow to expose parts of the
+Couchbase API directly, and rather perform extra conversion steps. Here are few
+examples:
+
+* JSON parser in PHP might silently convert integers in the documents to floats
+  with precision loss.
+* Increment/Decrement APIs in Couchbase operates on 64 bit numbers.
+* CAS values are reprensented as strings right now (but it would be more
+  efficient to use 64-number.
+* Expiration time is essentially 32-bit number, reprensenting Unix time in
+  seconds, so it has to be unsigned.
```

Comment:
these technical details maybe we have hide with some spoiler tag or mark them somehow that they will not look like known issues we are trying to solve. 

I think main idea here is that we do not test/support 32-bit x86 versions PHP SDK on any platform (Unix, MacOS and Windows)

Reply:
If we are not supporting 32-bit, should we give these details at all?

*I can hide them, it's just not something we've often done.

Did this come up from a customer actually wanting 32-bit?

Reply:
I agree that the details might be not necessary. I thought it might serve as justification why we no longer want to support it (in addition to the fact that it is hard to find modern OS that supports 32-bit).

So feel free to remove them.

Reply:
removed

Reply:
+1 hide the technical details